tests/python/pants_test/backend/jvm/tasks/BUILD

```diff
@@ -453,7 +453,7 @@ python_tests(
     'tests/python/pants_test:int-test',
   ],
   tags = {'integration'},
-  timeout = 240,
+  timeout = 540,
```

Comment:
did this really increase by more than twice the running time? or more of a ceiling?

Reply:
These are independent adjustments post https://pantsbuild.slack.com/archives/C15HUUSL9/p1564264147018100 ... some of these tests are just very highly variable. It didn't have anything to do with the change itself most likely.

Reply:
Since this will need another CI run anyway, I've broken these edits out over here: #8126